wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -233,6 +233,8 @@ public class ZMUserSession: NSObject {
         tornDown = true
     }
 
+    public var getUserClientFingerprint: GetUserClientFingerprintUseCase!
+
```

Comment:
Could this be lazy instead of implicitly unwrapped?

Reply:
I guess it could yes :)